torch/csrc/jit/pickler.cpp

```diff
@@ -242,8 +242,7 @@ void Pickler::pushStorageOfTensor(const at::Tensor& tensor) {
   pushString(tensor.device().str());
   // size
   pushInt(tensor.storage().size());
-  // view_metadata
-  push<PickleOpCode>(PickleOpCode::NONE);
```

Comment:
This wasn't used in `Unpickling` at all so this should be safe (it would be `arg.at(5)` around here https://github.com/pytorch/pytorch/blob/ac61adb5ef3ad193462d0a79a0594f8121f9b847/torch/csrc/jit/unpickler.cpp#L564-L573)